packages/cadl-ranch-specs/http/type/model/empty/main.tsp

```diff
@@ -0,0 +1,24 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Illustrates usage of empty model used in operation's parameters and responses.")
+@scenarioService("/type/model/empty")
+namespace Type.Model.Empty;
+
+@doc("Empty model used in operation parameters")
+model EmptyInput {}
+
+@doc("Empty model used in operation return type")
+model EmptyOutput {}
+
+@scenario
+@scenarioDoc("Send a POST request with the following body {}")
+@route("/input")
```

Comment:
missing `@post`?

Reply:
TSP compiler would automatically add @post here, I'll update the action explicitly.